packages/near-membrane-base/src/membrane.ts

```diff
@@ -1644,4 +1644,15 @@ export function createMembraneMarshall() {
     };
 }
 
-export const marshallSourceTextInStrictMode = `(function(){'use strict';return (${createMembraneMarshall.toString()})})()`;
+const TypeErrorCtor = TypeError;
+const marshallSourceTextInStrictMode = `(function(){'use strict';return (${createMembraneMarshall.toString()})})()`;
+// eslint-disable-next-line no-eval
+export function createConnector(evaluator: typeof eval) {
```

Comment:
I'm worry about adding more things to this membrane file. Membrane.ts is about the 1500 lines of the membrane.... which is supposed to be serializable... now adding more functionality here might imply that imports, and other things might be used that confuses the developer. I think we should just move this entire block to another file... maybe called connector.ts, in that file we import the marshall function and serialize it. So that way we can be confident that this file is about the serializable pieces.